lapis/src/main/kotlin/org/genspectrum/lapis/model/SiloQueryModel.kt

```diff
@@ -248,7 +250,10 @@ class SiloQueryModel(
                     type = sequenceType,
                     sequenceNames = mapSequenceNames(cleanedSequenceNames, sequenceType),
                     additionalFields = fastaHeaderTemplate.metadataFieldNames,
-                    orderByFields = mapSequenceOrderByFields(sequenceFilters.orderByFields, sequenceType),
+                    orderByFields = mapSequenceOrderByFields(
+                        sequenceFilters.orderByFields.toOrderByFields(),
+                        sequenceType,
+                    ).toOrderBySpec(),
```

Comment:
This seems odd - why do we need to convert to the list and then back to the spec?